tensorboard/plugins/debugger_v2/debug_data_multiplexer_test.py

```diff
@@ -34,11 +37,45 @@ def start(self):
 
 
 class DebuggerV2PluginTest(tf.test.TestCase):
-    def testRunInBackground(self):
-        mock_target = tf.compat.v1.test.mock.Mock()
-        with tf.compat.v1.test.mock.patch("threading.Thread", MockThread):
-            debug_data_multiplexer.run_in_background(mock_target)
-            mock_target.assert_called_once()
+    def testRunInBackgroundRepeatedly(self):
+        mock_target = mock.Mock()
+        sleep_state = {"count": 0}
+
+        def mock_sleep():
+            sleep_state["count"] += 1
+            if sleep_state["count"] == 10:
+                raise StopIteration()
+
+        with mock.patch("threading.Thread", MockThread), mock.patch.object(
+            debug_data_multiplexer._timer, "sleep", mock_sleep
+        ):
+            with self.assertRaises(StopIteration):
+                debug_data_multiplexer.run_repeatedly_in_background(mock_target)
+            self.assertEqual(mock_target.call_count, 10)
+
+
+class TimerTest(tf.test.TestCase):
```

Comment:
FWIW, see other comment, but this is basically just testing the `threading.Event` API so I'm not sure how much value it provides.

Reply:
These tests are removed together with the `Timer` class.